### ISSUE TITLE
Using boolean literals in default values for docker-compose.yml leads to build errors

### ISSUE THREAD

**Issue by juananpe** [codeforequity-at/botium-core#49]
It seems that [we should not use boolean literals (true, false)](https://devops.stackexchange.com/questions/537/why-is-one-not-allowed-to-use-a-boolean-in-a-docker-compose-yml) in docker-compose yaml configuration files. If we do so ([like here](https://github.com/codeforequity-at/botium-core/blob/4b606105554482b7e942a5d33be61cd7ccda6529/src/Defaults.js#L47))
docker-compose will yield an error:

```
botium-ProcessUtils Running Command: docker-compose -p core-CreateNewConversation -f /opt/botium/node_modules/botium-core/samples/botframework/botiumwork/core-CreateNewConversation-20180423-121846-Co3Q1/docker-compose.botium.yml -f /opt/botium/node_modules/botium-core/samples/botframework/botiumwork/core-CreateNewConversation-20180423-121846-Co3Q1/botframeworkmock/docker-compose.botframeworkmock.yml -f /opt/botium/node_modules/botium-core/samples/botframework/botiumwork/core-CreateNewConversation-20180423-121846-Co3Q1/docker-env.yml down +1ms
  botium-ProcessUtils docker-compose STDERR: The Compose file '/opt/botium/node_modules/botium-core/samples/botframework/botiumwork/core-CreateNewConversation-20180423-121846-Co3Q1/docker-env.yml' is invalid because:
  botium-ProcessUtils services.botium.environment.IS_BOTIUM_CONTAINER contains true, which is an invalid type, it should be a string, number, or a null

```

I would suggest to replace  `IS_BOTIUM_CONTAINER: true` by `IS_BOTIUM_CONTAINER: 'true'` instead.

**Comment by codeforequity-at** [comment]
I agree that the environment variables should be cleaned before handed over to docker container. On the same page, the capability configuration of Botium should become more robust to such kind of issues - should not matter if configuration for a flag says "true" or true or 'true' or 'yes' or y or 1 ...

**Comment by codeforequity-at** [comment]
Fixed with 1.3.11
